packages/zwave-js/src/lib/driver/Driver.ts

```diff
@@ -5345,6 +5345,7 @@ ${handlers.length} left`,
 	}
 
 	/** @internal */
+	// BUGBUG: do we need to replicate this for Long Range?
```

Comment:
Here's where the new command class could be used, I just didn't understand how it's supposed to be exercised if it never is advertised in any of the CC lists?